### PR TITLE
Unofficial dat added

### PR DESCRIPTION
**Pull request by marrikhong**
Hello, dat with unofficial stuff added; 16x FastRAM-compatible version; 3x cracked; 6x AmigaJay compilations. All tested.

### CODE REVIEW

File: dat/Commodore - CD32.dat [hunk 1]
```diff
@@ -0,0 +1,234 @@
+clrmamepro (
+	name "Commodore - CD32"
+	description "Commodore - CD32"
+	version "2024.11.23"
+	homepage "http://github.com/robloach/libretro-dats"
```
Comment (RobLoach) [inline dat/Commodore - CD32.dat:5]:
``` suggestion
	homepage "http://github.com/libretro/libretro-database"
```

Comment (marrikhong) [reply dat/Commodore - CD32.dat]: Thx, fixed.
If you know who manages the CD32 thumbnails please let me know. There's been a pull request hanging there for 3 weeks. If you guys don't like it or something just let me know what to fix/change. Thx!

Comment (RobLoach) [reply dat/Commodore - CD32.dat]: Thanks for the ping on that. Merged. Addded you to the libretro-thumbnails org too, so feel free to merge things that make sense for CD32.

Comment (marrikhong) [reply dat/Commodore - CD32.dat]: thx

**Pull request merged**